app/models/course.rb

```diff
@@ -140,8 +140,8 @@ def advance_start_at_duration_days=(value)
   def initialize_duplicate(duplicator, other)
     self.start_at += duplicator.time_shift
     self.end_at += duplicator.time_shift
-    self.title = duplicator.new_course_title
-    self.creator = duplicator.current_user
+    self.title = duplicator.params[:new_course_title]
```

Comment:
Options seems like a better name, let's use that. 

For documentation, we have some info when calling `duplicate_course`. Will see how else I can add info.